### PR TITLE
ignore certain types of errors

### PR DESCRIPTION
**Pull request by thousandsofthem**
Adds ability to ignore errors you do not want to log

### CODE REVIEW

**Comment by fazibear** [comment]
And what about tests for this feature ?

**Comment by fazibear** [comment]
What about test for this feature ?

**Comment by thousandsofthem** [comment]
Sorry, no idea how to test plug. any advices/examples?

File: lib/airbrakex/plug.ex [hunk 1]
```diff
@@ -16,12 +16,27 @@ defmodule Airbrakex.Plug do
           exception ->
             session = Map.get(conn.private, :plug_session)
 
-            Airbrakex.ExceptionParser.parse(exception)
-            |> Airbrakex.Notifier.notify([params: conn.params, session: session])
+            error = Airbrakex.ExceptionParser.parse(exception)
+            if proceed?(error) do
```
Comment (fazibear) [inline lib/airbrakex/plug.ex]: `proceed?(Application.get_env(:airbrakex, :ignore), error)` is better solution. Use pattern matching to match all cases and test this functions separately.

**Comment by thousandsofthem** [comment]
Thanks, implemented. Testing plug is still the issue, naive approach didn't work for me

**Comment by thousandsofthem** [comment]
@fazibear any updates here?

**Comment by joshuaswilcox** [comment]
Can we get this merged?

**Pull request merged**